apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx

```diff
@@ -1,9 +1,149 @@
 import { PatientSearchResult } from 'generated/graphql/schema';
+import styles from './PatientSearchResultListItem.module.scss';
+import { Link } from 'react-router-dom';
 
 type Props = {
     result: PatientSearchResult;
 };
 
-const PatientSearchResultListItem = ({ result }: Props) => <>{result.patient}</>;
+const PatientSearchResultListItem = ({ result }: Props) => {
+    const { legalName, birthday, gender, shortId, phones, names, emails, addresses, identification } = result;
+    const noData = 'No Data';
```

Comment:
question: should we use the existing <NoData /> component in here?

Reply:
Didn't realize there was an existing component, i'll take a look and see if it matches what we need here, but really this is just a string being stored so i didn't have to type "No data" a bunch of times.